querybook/webapp/components/DataDocDAGExporter/DataDocDAGExporterSettings.tsx

```diff
@@ -36,6 +38,15 @@ export const DataDocDAGExporterSettings: React.FunctionComponent<IProps> = ({
     return (
         <div className="DataDocDAGExporterSettings">
             <div className="DataDocDAGExporterSettings-form">
+                <FormField
+                    label="Use Templated Variables"
+                    className="horizontal-space-between"
+                >
+                    <ToggleSwitch
+                        checked={useTemplatedVariables}
+                        onChange={() => onSave(null, !useTemplatedVariables)}
```

Comment:
toggling this shouldn't save the meta? it should be something similar to handleSettingValuesChange